manager/manager.go

```diff
@@ -768,11 +783,44 @@ func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *c
 	return nil
 }
 
+// Enables overwriting an existing containerData/Handler object for a given containerName
+// createContainer just returns if a given containerName has a handler already
```

Comment:
@sjpotter Got you, can we add some comma or period, e.g. before `createContainer just...` ? cc @joshix for more suggestions :)